### DISCUSSION TITLE
Quid de l'intégration de l'OCS GE Ancienne Génération

### DISCUSSION THREAD

**Discussion by inesdartiguenave** [MTES-MCT/sparte#354] [General]
Nouveaux départements disponibles (couche occupation du sol + couche de diff) :

- [ ] Département 21 : 2010-2017
- [ ] Département 25 : 2010-2017
- [ ] Département 39 : 2010-2017
- [ ] Département 58 : 2010-2017
- [ ] Département 70 : 2010-2017
- [ ] Département 71 : 2010-2017
- [ ] Département 89 : 2010-2017
- [ ] Département 90 : 2010-2017
- [ ] Département 79 : 2010-2017
- [ ] Département 06 : 2004-2009-2014-2017

https://geoservices.ign.fr/ocsge#telechargement

Note : quid des départements avec 1 seul millésime ?

**Comment by alexisig** [comment]
Ces jeux de données ne sont pas du même type que ceux qu'on intègre habituellement : il s'agit de l'OCS GE "Ancienne génération". Jusqu'à présent nous avions la consigne de ne pas les utiliser. Est-ce que c'est quelque chose qui a changé ? 
Pour information, ces jeux de données n'ont pas de couche de diff, donc ils ne sont pas exploitables tel quel pour notre usage. Nous _pourrions_ calculer nous-même la couche de diff, mais ce serait un gros chantier.
Je renomme et passe l'US en discussion.
